base/stats_descriptions.go

```diff
@@ -343,6 +343,8 @@ const (
 	NumDocsPreFilterPublicAllDocsDesc = "The total number of documents returned on all public /_all_docs requests before filtering"
 
 	NumDocsPostFilterPublicAllDocsDesc = "The total number of documents returned on all public /_all_docs requests after filtering"
+
+	TombstoneCount = "The total number of tombstones processed by database"
```

Comment:
For consistency and clarity, suffix with `Desc` to indicate this is a description.

```suggestion
	TombstoneCountDesc = "The total number of tombstones processed by database"
```